docs/quickstart/services.rst

```diff
@@ -29,7 +27,7 @@ Forward the port used by Grafana:
 
 .. code-block:: shell
 
-   user@your-host $ kubectl --namespace metalk8s-monitoring port-forward svc/grafana 3000
+   user@your-host $ kubectl port-forward -n metalk8s-monitoring prometheus-operator-grafana-54b8dc5844-sf5x4 3000
```

Comment:
Using a Service is sufficient for port-forwarding. Also, we need to specify the kubeconfig to use (which was retrieved locally just before):

```suggestion
   user@your-host $ kubectl --kubeconfig=./admin.conf port-forward -n metalk8s-monitoring svc/prometheus-operator-grafana 3000:<local_port>
```

If you want, you could leave the default `<local_port>` to 3000.

Reply:
@gdemonet the thing grafana sevice port is now 80 (not anymore 3000) and you cannot use 80 for local port ... so what works on my env is the following: `kubectl --namespace metalk8s-monitoring port-forward svc/prometheus-operator-grafana 3000:80 --kubeconfig=/tmp/admin.conf`. works for you ?